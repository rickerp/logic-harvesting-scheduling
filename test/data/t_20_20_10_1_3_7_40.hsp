40
3
8 3 5 5 4 3 8 6 8 5 7 6 5 4 1 10 5 6 4 10 9 3 3 10 2 4 10 7 2 2 8 4 6 1 3 6 3 5 8 1
8 2 4 7 10 12 13 19 23
8 1 3 6 16 17 18 22 31
6 2 6 16 17 18 22
7 1 5 7 8 10 12 13
3 4 6 8
4 2 3 5 9
4 1 4 8 10
9 4 5 7 9 10 13 14 20 33
8 6 8 10 13 14 15 27 32
11 1 4 7 8 9 12 13 14 19 20 32
2 12 19
9 1 4 10 11 13 19 23 28 34
10 1 4 8 9 10 12 14 19 20 36
10 8 9 10 13 15 20 32 33 37 39
6 9 14 16 20 32 36
12 2 3 15 17 19 20 27 32 33 34 36 37
10 2 3 16 18 20 27 28 32 34 36
4 2 3 17 21
9 1 10 11 12 13 16 20 24 39
18 8 10 13 14 15 16 17 19 21 24 25 27 32 33 34 36 38 39
8 18 20 22 24 25 26 28 34
7 2 3 21 24 25 26 28
7 1 12 24 28 34 36 39
14 19 20 21 22 23 25 27 28 32 33 34 36 37 39
7 20 21 22 24 26 27 29
6 21 22 25 27 29 31
13 9 16 17 20 24 25 26 28 29 32 33 34 40
12 12 17 21 22 23 24 27 29 32 34 36 39
5 25 26 27 28 30
5 29 31 33 34 40
6 2 26 30 33 38 40
15 9 10 14 15 16 17 20 24 27 28 33 34 36 37 39
12 8 14 16 20 24 27 30 31 32 35 37 40
13 12 16 17 20 21 23 24 27 28 30 32 35 36
7 33 34 36 37 38 39 40
13 13 15 16 17 20 23 24 28 32 34 35 37 39
10 14 16 24 32 33 35 36 38 39 40
6 20 31 35 37 39 40
12 14 19 20 23 24 28 32 35 36 37 38 40
8 27 30 31 33 35 37 38 39
6 4 4 5 4 5 4 6 2 6 5 4 6 6 2 1 2 2 5 6 1 4 5 5 4 5 4 4 4 4 4 3 5 5 5 6 7 6 5 1
5 2 3 1 7 4 1 7 6 3 5 4 4 2 1 1 4 2 4 1 3 5 1 7 3 3 6 2 1 3 1 5 2 3 4 6 6 4 5 2
7 7 5 3 7 3 1 3 5 3 1 5 7 6 5 7 1 1 7 2 1 7 4 2 7 7 5 3 2 1 5 6 5 7 6 2 1 7 5 3
1
