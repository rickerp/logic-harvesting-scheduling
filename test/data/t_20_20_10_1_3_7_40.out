138
10 4 6 14 18 19 22 23 29 31 35
9 1 3 8 11 21 27 30 36 38
9 2 5 7 13 15 26 33 34 39
1 37
